core/assets/bundles/bundle.properties

```diff
@@ -1240,78 +1240,139 @@ category.optional = Optional Enhancements
+setting.mouseposition.description = Displays the mouse cursor's coordinates on the screen.
 setting.musicvol.name = Music Volume
+setting.musicvol.description = Adjusts the volume of the in-game music.
 setting.atmosphere.name = Show Planet Atmosphere
+setting.atmosphere.description = Toggles the rendering of the planet's atmosphere in the background.
 setting.drawlight.name = Draw Darkness/Lighting
+setting.drawlight.description = Enables or disables dynamic lighting and darkness effects.
 setting.ambientvol.name = Ambient Volume
+setting.ambientvol.description = Adjusts the volume of ambient sounds.
 setting.mutemusic.name = Mute Music
+setting.mutemusic.description = Mutes all in-game music.
 setting.sfxvol.name = SFX Volume
+setting.sfxvol.description = Adjusts the volume of sound effects.
 setting.mutesound.name = Mute Sound
+setting.mutesound.description = Mutes all in-game sounds.
 setting.crashreport.name = Send Anonymous Crash Reports
+setting.crashreport.description = When enabled, anonymous crash reports will be sent to the developers to help improve the game.
 setting.communityservers.name = Fetch Community Server List
+setting.communityservers.description = When enabled, the game will fetch and display a list of community-run multiplayer servers.
 setting.savecreate.name = Auto-Create Saves
+setting.savecreate.description = When enabled, the game will automatically create a new save file when starting a new game.
 setting.steampublichost.name = Public Game Visibility
+setting.steampublichost.description = Controls who can see and join your game when hosting on Steam.
 setting.playerlimit.name = Player Limit
+setting.playerlimit.description = Sets the maximum number of players that can join your server.
 setting.chatopacity.name = Chat Opacity
+setting.chatopacity.description = Adjusts the transparency of the chat window.
 setting.lasersopacity.name = Power Laser Opacity
+setting.lasersopacity.description = Adjusts the opacity of power node lasers.
 setting.unitlaseropacity.name = Unit Mining Beam Opacity
+setting.unitlaseropacity.description = Adjusts the opacity of unit mining beams.
 setting.bridgeopacity.name = Bridge Opacity
+setting.bridgeopacity.description = Adjusts the opacity of bridge conveyors.
 setting.playerchat.name = Display Player Bubble Chat
+setting.playerchat.description = When enabled, chat messages from players will appear in bubbles above their heads.
 setting.showweather.name = Show Weather Graphics
+setting.showweather.description = Toggles the display of weather effects like rain and snow.
 setting.hidedisplays.name = Hide Logic Displays
+setting.hidedisplays.description = Hides the content of logic displays, which may improve performance.
 setting.macnotch.name = Adapt interface to display notch
-setting.macnotch.description = Restart required to apply changes
+setting.macnotch.description = On macOS devices with a display notch, this setting adjusts the UI to avoid it.
```

Comment:
you forgot to read the code, these setting requires to restart to apply. I'm not Mac user, so you can compromise my review